adafruit_onewire/bus.py

```diff
@@ -0,0 +1,206 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2017 Carter Nelson for Adafruit Industries
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+"""
+`OneWire Bus`
+====================================================
+
+Provide access to a 1-Wire bus.
+
+* Author(s): Carter Nelson
+"""
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_OneWire.git"
+
+import busio
+from micropython import const
+
+_SEARCH_ROM = const(0xF0)
+_MATCH_ROM = const(0x55)
```

Comment:
if you do `_MATCH_ROM = b'\x55'`instead, you won't need to convert it to `bytearray` everywhere.

Reply:
thanks, those converts were kind of kluge leftover from changing all calls to be general purpose and pass in arrays.

Reply:
i changed in the other module, in this one there are no converts used